AIDevGallery/ProjectGenerator/Generator.cs

```diff
@@ -674,10 +674,16 @@ private string CleanXamlSource(string xamlCode, string newNamespace, out string
         if (match.Success)
         {
             var oldClassFullName = match.Groups[1].Value;
-            _ = oldClassFullName[..oldClassFullName.LastIndexOf('.')];
             className = oldClassFullName[(oldClassFullName.LastIndexOf('.') + 1)..];
 
-            xamlCode = xamlCode.Replace(match.Value, @$"x:Class=""{newNamespace}.Sample""");
+            if (oldClassFullName.Contains(".SharedCode."))
+            {
+                xamlCode = xamlCode.Replace(match.Value, @$"x:Class=""{newNamespace}.{className}""");
```

Comment:
What does this do precisely? Is it because of the rename/move?

Reply:
done

Reply:
There is a bug when a usercontrol is exported, the class is renamed to .Sample instead of the usercontrol class name. This fixed that